src/power_divergence.jl

```diff
@@ -119,31 +120,24 @@ function ci_sison_glaz(x::PowerDivergenceTest, alpha::Float64; skew_correct::Boo
             lambda = x.observed[i]
             #run moments
             a = lambda + c
-            b = max(lambda - c, 0)
-            if lambda > 0
```

Comment:
Is `lambda` always positive?

Reply:
Yes, data is validated in PowerDivergenceTest function (x.observed[i] needs to be non-negative)